features/recognize.js

```diff
@@ -29,11 +29,15 @@ async function respondToRecognitionMessage({ message, client }) {
   });
   let gratitude;
   try {
+    const giver = await userInfo(client, message.user);
     gratitude = {
-      giver: await userInfo(client, message.user),
+      giver: giver,
       receivers: await Promise.all(
         recognition
           .gratitudeReceiverIdsIn(message.text)
+          .filter(async function (receiver) {
+            return receiver != giver;
+          })
```

Comment:
```suggestion
          .filter((receiver) => receiver !== giver)
```

Reply:
Leaving out the `async` works here? Cause I did try to go with an arrow function for the filter and somehow it didn't work as expected so I had to go with a traditional `function` call

Reply:
yes, `async` is only needed if you want to use `await` within the function body

Reply:
for example, the `async` right below this change is also not needed

Reply:
Fair enough, for some reason my brain thought it actually was an `await` lol

Reply:
the arrow function vs function keyword is just stylistic in this case, but I pretty much exclusively use arrow functions.  the key difference is the autobinding of the `this` context, which only really matters if you're using classes

Reply:
Yeah I tend to only ever use arrow functions as well (because  not worried about the scope of `this`). Not sure why it wasn't working properly when I was testing at first.